hedera-node/hedera-app/src/main/java/module-info.java

```diff
@@ -99,4 +99,7 @@
             com.swirlds.common,
             com.swirlds.platform;
     exports com.hedera.node.app.validation;
+    exports com.hedera.node.app.state.listeners to
+            com.hedera.node.app.test.fixtures,
+            com.swirlds.common;
```

Comment:
As best I can tell `com.swirlds.common` doesn't use anything from this module (which is good, given this module `requires transitive com.swirlds.common` on line 17).
We should remove all exports to `com.swirlds.common` here to eliminate the potential for confusion.
`com.swirlds.platform.core` appears to be the same.
Removing both of those significantly simplifies this file.

Reply:
Yes @jsync-swirlds . Thank you. Will change

Reply:
Fixed. Thanks @jsync-swirlds